tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -857,14 +857,14 @@ void HWCommandQueue::enqueue_command(T& command, bool blocking) {
 
 // TODO: Currently converting page ordering from interleaved to sharded and then doing contiguous read/write
 //  Look into modifying command to do read/write of a page at a time to avoid doing copy
-void convert_interleaved_to_sharded_on_host(const void* host, const Buffer& buffer, bool read = false) {
+void * convert_interleaved_to_sharded_on_host(const void* host, const Buffer& buffer, bool read = false) {
     const uint32_t num_pages = buffer.num_pages();
     const uint32_t page_size = buffer.page_size();
 
     const uint32_t size_in_bytes = num_pages * page_size;
 
-    void* temp = malloc(size_in_bytes);
-    memcpy(temp, host, size_in_bytes);
+    void* swapped = malloc(size_in_bytes);
+    memcpy(swapped, host, size_in_bytes);
```

Comment:
You don't need this memcpy here

Reply:
ah you're right, still an artifact of before :p